README.md

```
@@ -133,23 +199,35 @@ $ hivemind-core delete-client 1
-  useful in scenarios where certain clients should only perform limited actions (e.g., making another device speak via
-  TTS).
+  Use this command to enable certain message types, particularly when extending a client’s communication capabilities (e.g., allowing TTS commands).
+
+---
+
+### `blacklist-msg`
+
+Block specific message types from being sent by a client.
+
+```bash
+$ hivemind-core allow-msg "speak"
```

Markdownlint (0.37.0)

Dollar signs used before commands without showing output

(MD014, commands-show-output)